code/modules/reagents/machinery/chem_master.dm

```diff
@@ -10,29 +10,25 @@
 	use_power = 1
 	idle_power_usage = 20
 	var/obj/item/weapon/reagent_containers/glass/beaker = null
-	var/obj/item/weapon/storage/pill_bottle/loaded_pill_bottle = null
 	var/mode = 0
 	var/condi = 0
 	var/useramount = 30 // Last used amount
 	var/pillamount = 10
 	var/bottlesprite = "bottle-1" //yes, strings
 	var/pillsprite = "1"
 	var/client/has_sprites = list()
-	var/max_pill_count = 20
+	var/max_pill_count = 24
 	reagent_flags = OPENCONTAINER
 
 /obj/machinery/chem_master/RefreshParts()
-	if(!reagents)
+	/*if(!reagents)
 		create_reagents(10)
 	reagents.maximum_volume = 0
 	for(var/obj/item/weapon/reagent_containers/glass/G in component_parts)
 		reagents.maximum_volume += G.volume
-		G.reagents.trans_to_holder(reagents, G.volume)
+		G.reagents.trans_to_holder(reagents, G.volume)*/
```

Comment:
Is this going to be re-implemented later or something? Because you deleted all the other things here instead of commenting them out.

Reply:
Yeah, i left it so it can be used in future in some other way.
So this will save the trouble.